app/assets/stylesheets/cclow/_latest-additions-slider.scss

```diff
@@ -0,0 +1,78 @@
+@import "variables";
+
+.slider-container {
+  .slick-slide {
+    padding: 0 10px;
+  }
+
+  .slider {
+    margin: 0 -10px;
```

Comment:
does this `10px` depend on the 10px in `slick-slide` class? meaning: if I change it to -15 for example, will the slick-slide need to be changed too? if so, please move it to a variable